scripts/arc/documentation/arcdocs.md

```diff
@@ -0,0 +1,270 @@
+
+- [Bash version 4](https://www.gnu.org/software/bash/)  is required if configuring proxy endpoint via Bash.
+
+
+
+## Identify workspace resource ID
+
+To enable monitoring of your cluster with an existing Log Analytics workspace, perform the following steps to first identify the full resource ID of your Log Analytics workspace. This is required for the `logAnalyticsWorkspaceResourceID` parameter when you run the command to enable the monitoring extension against the specified workspace.  This information can be found in the _Overview_ blade of your Log Analytics workspace through the Azure Portal or using the following steps via CLI.
+
+1. List all the subscriptions that you have access to using the following command:
+
+    ```azurecli
+    az account list --all -o table
+    ```
+
+    The output will resemble the following:
+
+    ```azurecli
+    Name                                  CloudName    SubscriptionId                        State    IsDefault
+    ------------------------------------  -----------  ------------------------------------  -------  -----------
+    Microsoft Azure                       AzureCloud   0fb60ef2-03cc-4290-b595-e71108e8f4ce  Enabled  True
+    ```
+
+    Copy the value for **SubscriptionId**.
+
+2. Switch to the subscription hosting the Log Analytics workspace using the following command:
+
+    ```azurecli
+    az account set -s <subscriptionId of the workspace>
+    ```
+
+3. The following example displays the list of workspaces in your subscriptions in the default JSON format.
+
+    ```
+    az resource list --resource-type Microsoft.OperationalInsights/workspaces -o json
+    ```
+
+    In the output, find the workspace name, and then copy the full resource ID of that Log Analytics workspace under the field **ID**.
+
+
+## Migrate to the Extension Model
+
+If you had previously deployed Azure Monitor for containers on this cluster using Helm directly without extensions, follow the instructions listed [here](https://docs.microsoft.com/azure/azure-monitor/insights/container-insights-optout-hybrid) to delete this Helm chart. Once the deletion is done, you can then proceed to subsequent sections on creating an extension instance for Azure Monitor for containers.
+
+
+### Create Azure Monitor extension instance
+
+Please check the [prerequisites](https://docs.microsoft.com/azure/azure-monitor/insights/container-insights-enable-arc-enabled-clusters#prerequisites) to verify you have the required permissions to enable container insights extension on your Azure Arc enabled Kubernetes clusters.
+
+#### Onboarding via Portal
+
+1.) In the Azure Portal, select the Arc enabled Kubernetes cluster that you wish to monitor.
+
+2.) Select the Insights (preview) item under the Monitoring section
+![](./Images/ArcResourceBlade.png)
+
+3.) On the onboarding page, select the Configure Azure Monitor button
```

Comment:
We need to take latest onboarding screenshots since the proxy option will be removed. And also we need to add the screenshot to enable the onboarding from Multi-cluster Ux also.